packages/keystatic/src/form/api.tsx

```diff
@@ -292,7 +292,6 @@ export type ComponentBlock<
       chromeless?: false;
       toolbar?: (props: {
         props: Record<string, any>;
-        onShowEditMode(): void;
```

Comment:
Could you add this prop back since we probably don't want to do a breaking change for this? You'll probably want to use the [`DialogContainer`](https://keystar-ui.vercel.app/package/dialog/dialog-container) rather than `DialogTrigger` component so that you can manage the state yourself